docs/api-reference/miscellaneous/scalars/metadata.mdx

```diff
@@ -77,10 +77,10 @@ Metadata is a map of key-value pairs, both keys and values are `String`.
 Example:
 
 ```
```

Comment:
Wouldn't it have worked if we did this instead?
```suggestion
```json
```
```

Reply:
It doesn't work. And I think it's benenficial to replace unicode with curly braces for readability in any case.

In addition, specifying the code block as json would provide syntax highlighting, as below
<img width="647" alt="image" src="https://github.com/saleor/saleor-docs/assets/30428258/7fe373d3-5d00-4aff-8a72-ff49b2bba388">

Without specifying json, the code block would look like below
<img width="641" alt="image" src="https://github.com/saleor/saleor-docs/assets/30428258/fb22d716-d8f1-487d-ab7c-2aa78b48e231">

Reply:
I looked through the project and found that json code blocks have syntax highlighting. So I added json to the metadata code blocks.